dev/deps/spark-deps-hadoop-3-hive-2.3

```diff
@@ -21,6 +21,11 @@ arrow-memory-core/18.0.0//arrow-memory-core-18.0.0.jar
 arrow-memory-netty-buffer-patch/18.0.0//arrow-memory-netty-buffer-patch-18.0.0.jar
 arrow-memory-netty/18.0.0//arrow-memory-netty-18.0.0.jar
 arrow-vector/18.0.0//arrow-vector-18.0.0.jar
+asm-analysis/9.2//asm-analysis-9.2.jar
+asm-commons/9.2//asm-commons-9.2.jar
+asm-tree/9.2//asm-tree-9.2.jar
+asm-util/9.2//asm-util-9.2.jar
+asm/9.2//asm-9.2.jar
```

Comment:
Spark  master branch is currently using `asm` version 9.7.1, and I think we should unify to version 9.7.1.